tests/test_selenium.py

```diff
@@ -0,0 +1,144 @@
+
+
+@pytest.mark.regression
+@pytest.mark.parametrize("navigation_button", ["mountain", "road", "gravel", "shops", "sale", "electric_bikes"])
+def test_redirect_to_navigation_blocks(create_driver, navigation_button):
+    driver = create_driver
+    main_page = MainPage(driver)
+
+    if navigation_button == "mountain":
+        main_page.check_mountain_bikes_block()
+    elif navigation_button == "road":
+        main_page.check_road_bikes_block()
+    elif navigation_button == "gravel":
+        main_page.check_gravel_bikes_block()
+    elif navigation_button == "shops":
+        main_page.click_shops_button()
+        assert ShopsPage(driver).check_if_shops_is_loaded(), "shop page is not displayed normally"
+    elif navigation_button == "sale":
+        main_page.click_sale_button()
+        assert SalePage(driver).check_if_sales_page_is_opened(), "sale page is not displayed properly"
+    elif navigation_button == "electric_bikes":
+        main_page.open_electric_bikes()
+        assert ElectricBikesPage(driver).check_electric_bikes_page(), "electric bikes page is not opened properly"
+
+
+@pytest.mark.smoke
+def test_view_all_stores(create_driver):
+    driver = create_driver
+    MainPage(driver).click_shops_button().geo_warning_check().click_open_all_stores()
+    assert ShopsPage.check_if_all_stores_is_visible, "Stores are failed to load"
+
+
+@pytest.mark.regression
+def test_open_store_in_appiano(create_driver):
+    driver = create_driver
+    MainPage(driver).click_shops_button().geo_warning_check().click_open_all_stores().open_italy_retailers(). \
+        open_appiano_service_center()
+    assert ShopsPage.check_if_appiano_store_is_loaded, "Store in Appiano is failed to load"
+
+
+@pytest.mark.smoke
+@pytest.mark.parametrize("sale_item", ["bikes", "equipments", "apparel", "clearance_centers"])
+def test_open_all_sale_items(create_driver, sale_item):
+    driver = create_driver
+    sale_page = MainPage(driver).click_sale_button()
+
+    if sale_item == 'bikes':
+        sale_page.open_all_sale_bikes()
+        assert sale_page.check_if_bikes_sales_is_opened(), 'not a page with bikes is opened, smth is wrong'
+        assert sale_page.check_if_sales_filter_is_on(), 'sale filter is not ok'
+    elif sale_item == 'equipments':
+        sale_page.open_all_sale_equipment()
+        assert sale_page.check_if_all_sale_equipment_is_opened(), 'not a page with sale equipments is opened'
+        assert sale_page.check_if_sales_filter_is_on(), 'sale filter is broken'
+    elif sale_item == 'apparel':
+        sale_page.open_all_sale_apparel()
+        assert sale_page.check_if_all_sale_apparel_is_opened(), 'not a page with bike cloth is opened'
+        assert sale_page.check_if_sales_filter_is_on(), 'sale filter is set improperly'
+    elif sale_item == 'clearance_centers':
+        sale_page.open_track_clearance_centers()
+        assert sale_page.check_if_clearance_centers_opened()
```

Comment:
same here. it is not a good practice to have if statements in tests. Parametrization should be used when you have completely same code but with different inputs. In you case you have different TCs.